packages/svelte/src/compiler/phases/3-transform/client/visitors/shared/fragment.js

```diff
@@ -150,6 +150,12 @@ function is_static_element(node) {
 			return false;
 		}
 
+		// an img with loading lazy is not static but needs the `img` to be
+		// generated to be passed to `handle_lazy_img`
```

Comment:
I couldn't understand this comment without reading through the code as it assumes some familiarity with why `handle_lazy_img` is needed. How about something like this instead?
```suggestion
		// We need to apply src and loading after appending the img to the DOM for lazy loading to work
```

Reply:
Yup much better